api/client/formatter/custom.go

```diff
@@ -142,6 +143,20 @@ func (c *containerContext) Label(name string) string {
 	return c.c.Labels[name]
 }
 
+func (c *containerContext) Mounts() string {
+	c.addHeader(mountsHeader)
+
+	var mounts []string
+	for _, m := range c.c.Mounts {
+		name := m.Name
+		if c.trunc {
+			name = stringutils.Truncate(name, 15)
```

Comment:
kinda magic. usually see 12 for Container IDs. I'm not saying to change it, I just wonder if there was specific reason it was chosen that could be written down. "not too short but not too long" is good enough for me.